docs/snippets/vue/loader-story.3.js.mdx

```diff
@@ -0,0 +1,28 @@
+```js
+// TodoItem.stories.js
+
+import TodoItem from './TodoItem.vue';
+
+import fetch from 'node-fetch';
+
+export default {
+  component: TodoItem,
+  title: 'Examples/Loader',
+};
+
+export const Primary = (args, { loaded: { todo } }) => {
+  return {
+    components: { TodoItem },
+    setup() {
+      return { args, todo: todo };
+    },
+    template: `<TodoItem :todo="todo"/>`,
+  };
+};
+
+SampleStory.loaders = [
```

Comment:
Primary.loaders?

Reply:
Fixed with the latest commit. Marking this as solved